### DISCUSSION TITLE
Jump labels annotation

### DISCUSSION THREAD

**Discussion by edward-s** [folke/flash.nvim#180] [Q&A]
Assuming my cursor is at this position
<img width="457" alt="Screenshot 2023-07-24 at 10 09 46 AM" src="https://github.com/folke/flash.nvim/assets/237278/e0ffdcf4-1c17-45b8-8d72-4b48a673595f">

i want to reach `person2Seeder` at the bottom of the screen
and i type `s` `se`
<img width="451" alt="Screenshot 2023-07-24 at 10 10 02 AM" src="https://github.com/folke/flash.nvim/assets/237278/81d11e14-9240-4744-ac46-498827ee790a">

 `person2Seeder` is highlighted in green, but I can't reach it cos there is no jump label, is there any way to go to annotate all the targets with jump labels?

**Comment by folke** [comment]
nope

**Reply by edward-s** [reply]
ok, so is the correct approach to move down closer to target before searching?
